### PR TITLE
Create Fresh.js

### PR DESCRIPTION
**Pull request by ItzJonax**

### CODE REVIEW

**Review by callumok2004 — CHANGES_REQUESTED** [review]
Fresh isnt even open.. close this

**Comment by callumok2004** [comment]
the site is up as a joke

**Comment by ItzJonax** [comment]
> Fresh isnt even open.. close this

what? yes it is i literally set up everything for them

File: services/Fresh.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const tags = require('../tags'),
+  langs = require('langs')
+
+module.exports = () => {
+  return {
+    "name": "Fresh",
+    "aliases": ["fresh", "freshradio", "fr"],
```
Comment (callumok2004) [inline services/Fresh.js:7]: remove the name as an alias

File: services/Fresh.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "logo": "https://freshradio.pw/images/FRESH%20NEW.png",
+    "id": "13d4a520-3329-4afb-9292-c9356c67b7cf",
```
Comment (callumok2004) [inline services/Fresh.js:9]: i hope this is not the same as any other!

**Review by callumok2004 — CHANGES_REQUESTED** [review]

File: services/Fresh.js (hunk 2 above)
Comment (PiggyPlex) [reply services/Fresh.js:9]: fr

**Review by PiggyPlex — APPROVED** [review]
ofc i approve of my own changes !!

**Pull request merged**